components/raftstore/src/store/fsm/store.rs

```diff
@@ -1285,6 +1287,26 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
         let raftlog_gc_scheduler = workers
             .background_worker
             .start_with_timer("raft-gc-worker", raftlog_gc_runner);
+        let router_clone = self.router();
+        let engines_clone = engines.clone();
+        workers.purge_worker.spawn_interval_task(
```

Comment:
move `raft-gc-worker` out of background worker instead of adding a new worker.

Reply:
They should be in different threads, 1min+ purge job will block gc, causing OOM.